packages/roosterjs-editor-dom/lib/metadata/metadata.ts

```diff
@@ -57,9 +57,22 @@ export function setMetadata<T>(element: HTMLElement, metadata: T, def?: Definiti
 }
 
 /**
- * Remove metadata from the given element if any
+ * Remove specific metadata from the given element if any
  * @param element The element to remove metadata from
+ * @param metadata The metadata tag to remove
  */
-export function removeMetadata(element: HTMLElement) {
+export function removeMetadata<T>(element: HTMLElement, metadata: string) {
```

Comment:
This is a breaking change. For new added parameter, we need to make it optional, and when it is not passed, the function should work in the same way with the code before change.

Otherwise, other people who is using this function will be broken

Reply:
Does that mean I can never change function names if they are exported?

Reply:
Changing a function name means we need to bump major version (v8 => v9). We don't want this happen just because we want to rename a function.

Reply:
Sometimes we do want to rename a function, e.g. the original function name has a typo. In that case, will still keep the old one and mark it as deprecated. For example:

```ts
export function newFunc() { ... }

/**
 * @deprecated Use newFunc instead
 */
export const oldFunc = newFunc;
```

In that way, people who don't want to make change to their code can still upgrade version without breaking the build.